src/main/java/fact/io/hdureader/FITSStream.java

```diff
@@ -43,7 +43,9 @@ public void init() throws Exception {
         //get calibration constants they are stored in the first (and only) row of this hdu.
         HDU offsetHDU = fits.getHDU("ZDrsCellOffsets");
         if (offsetHDU != null){
-            OptionalTypesMap<String, Serializable> row = ZFITSHeapReader.forTable(offsetHDU.getBinTable()).getNextRow();
+            ZFITSHeapReader reader = ZFITSHeapReader.forTable(offsetHDU.getBinTable());
+            boolean ignoreWrongTileHeader;
```

Comment:
I was just confused, what is happening in java, if you assign something inside a function call? 

I expected this:
```java
boolean ignoreWrongTileHeader = true;
OptionalTypesMap<String, Serializable> row = reader.getNextRow(ignoreWrongTileHeader);
```

This assignment inside the function call confused me.